src/main/java/uk/gov/hmcts/reform/orgrolemapping/domain/model/UserProfile.java

```diff
@@ -25,52 +30,58 @@
     //@JsonFormat(pattern = "YYYY-MM-dd HH:mm:ss:SSS")
     private LocalDateTime createdTime;
     //@JsonFormat(pattern = "YYYY-MM-dd HH:mm:ss:SSS")
-    private LocalDateTime lastUpdateTime;
+    private LocalDateTime lastUpdatedTime;
```

Comment:
we can remove the createdTime and lastUpdatedTime from our POJO as there is no usage of these fields in our flow.

Reply:
Will remove it in https://github.com/hmcts/am-org-role-mapping-service/pull/64/files